experimental/dds/tree2/src/feature-libraries/default-field-kinds/optionalField.ts

```diff
@@ -105,42 +218,50 @@ export const optionalChangeRebaser: FieldChangeRebaser<OptionalChangeset> = {
 		invertChild: NodeChangeInverter,
 		reviver: NodeReviver,
 	): OptionalChangeset => {
-		const inverse: OptionalChangeset = {};
+		// Changes to the child that existed in this field before `change` was applied.
+		let originalChildChanges: NodeChangeset | undefined;
+		const inverseChildChanges = new ChildChangeMap<NodeChangeset>();
+		if (change.childChanges !== undefined) {
+			for (const [id, childChange] of change.childChanges) {
+				if (id === "self" && change.fieldChange !== undefined) {
+					originalChildChanges = invertChild(childChange, 0);
+				} else {
+					inverseChildChanges.set(
+						// This makes assumptions about how sandwich rebasing works
```

Comment:
Please document what the assumption is.

Reply:
@Abe27342 is the assumption here about the id being valid and that it should maybe be a rollback id? Want to update this comment before merging